public/app/plugins/panel/canvas/ConnectionSVG.tsx

```diff
@@ -23,7 +24,9 @@ export const ConnectionSVG = ({ setSVGRef, setLineRef, scene }: Props) => {
   const styles = useStyles2(getStyles);
 
   const CONNECTION_LINE_ID = 'connectionLineId';
-  const CONNECTION_HEAD_ID = 'head';
+
+  const id = useUniqueId();
+  const CONNECTION_HEAD_ID = `head-${id}`;
```

Comment:
```suggestion
  const CONNECTION_HEAD_ID =  useMemo( () =>`head-${counter++}`, [] );
```
rather than the kinda complicated import, maybe just add a counter?  this does not need to be a real UUID

Reply:
sad counter added!

Reply:
@ryantxu testing this more, I think because of the way we are rendering when switching between edit/view mode we end up again with conflicting ids, making the arrows disappear (not just the heads)...

![conflicting-ids](https://user-images.githubusercontent.com/88068998/213813797-43daacec-aa0f-4471-aa59-1bd880e9a0c6.png)

Reply:
can start with `time.Now()`?